nir/src/main/scala/scala/scalanative/nir/Ops.scala

```diff
@@ -46,6 +49,9 @@ object Op {
   final case class Conv(conv: nir.Conv, ty: Type, value: Val)     extends Pure
   final case class Select(cond: Val, thenv: Val, elsev: Val)      extends Pure
 
+  final case class Box(code: Char, obj: Val)   extends Op
+  final case class Unbox(code: Char, obj: Val) extends Op
```

Comment:
1. Those ops belong to the high-level group of ops below.
2. They should take a box type, and not a character code.